File: ssaExample/ssa_Package.java
import java.util.Map;

public class ssa_Package {

	public ssa_Program Prog;
	public types_Package Pkg;
	public Map<String, Object> Members;
	public Map<Object, Object> objects;
	public ssa_Function init;
	public Boolean debug;
	public Boolean syntax;
	public sync_Once buildOnce;
	public int ninit;
	public types_Info info;
	public ast_File[] files;
	public ssa_Function[] created;
	public Map<Object, String> initVersion;
}

File: ssaExample/ssa_Package.kt
class ssa_Package {

	var Prog: ssa_Program? = null
	var Pkg: types_Package? = null
	var Members: Map<String, Any>? = null
	var objects: Map<Any, Any>? = null
	var init: ssa_Function? = null
	var debug: Boolean? = null
	var syntax: Boolean? = null
	var buildOnce: sync_Once? = null
	var ninit: Int? = null
	var info: types_Info? = null
	var files: Array<ast_File>? = null
	var created: Array<ssa_Function>? = null
	var initVersion: Map<Any, String>? = null
}
